docs/testing-abstractions.md

```
@@ -0,0 +1,130 @@
+use ironclaw::testing::worker_harness::{make_worker, TerminalMethod};
+
+let worker = make_worker(vec![]).await;
+TerminalMethod::Completed.apply_transition(&worker).await;
+```
+
+**When to use:** Use the worker harness when testing `Worker` behavior
+specifically.
+
+## Choosing the Right Abstraction
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)